.github/CODEOWNERS

```diff
@@ -98,6 +119,7 @@ src/applications/caregivers @department-of-veterans-affairs/1010-health-apps-fro
 
 src/applications/hca @department-of-veterans-affairs/1010-health-apps-frontend @department-of-veterans-affairs/benefits-team-1-frontend
 src/applications/ezr @department-of-veterans-affairs/1010-health-apps-frontend @department-of-veterans-affairs/benefits-team-1-frontend
+src/applications/static-pages/letters-mobile-cta @department-of-veterans-affairs/1010-health-apps-frontend
```

Comment:
Pivot! https://dsva.slack.com/archives/C52CL1PKQ/p1695232851416249?thread_ts=1695156355.685709&cid=C52CL1PKQ. Moves back to PW.

Reply:
Updated! Thank you